app/assets/stylesheets/application.css.scss

```diff
@@ -90,7 +90,15 @@ a, a:visited, a:link {
 }
 
 .flash {
-    text-align: center;
+  text-align: center;
+}
+
+.flash_notice {
+  font-size: 1.25em;
```

Comment:
Properties should be ordered -moz-box-shadow, -webkit-box-shadow, box-shadow, font-size, padding